modules/installation-and-upgrade/pages/container-deployment/suma/server-deployment-vmdk-suma.adoc

```diff
@@ -0,0 +1,315 @@
+
+You need to hash the root password, for example, by using the [command]``openssl`` command:
+
+----
+openssl passwd -6
+----
+
+The command creates a hash of the password you chose.
+Use this hash as the value of the [literal]``password`` attribute.
+
+Prepare the needed configuration files using the following commands:
+
+----
+touch network-config
+touch meta-data
+----
+
+Create a file named [path]``user-data`` with the following content:
+
+----
+#cloud-config
+chpasswd:
+  expire: false
+  users:
+    - name: root
+      password: $2a$10$qV298UV11u9lCFDjpHpCUe1cErBiVR.G3shukxs3.2PAO1xhJWs0K
+----
+
+Prepare [systemitem]``Cloud Init`` ISO file using the command:
+
+----
+mkisofs -rational-rock -joliet -o suma_cloudinit.iso -V cidata network-config meta-data user-data
+----
+
+Attach the created [path]``suma_cloudinit.iso`` file as a volume to the creating virtual machine.
+This particular example is setting [systemitem]``root`` password to `linux`.
+Substitute your password hash for the one in this example
+
+
+
+
+[[quickstart.sect.vmware.settings]]
+== {productname} Virtual Machine Settings - VMware
+
+This sections describes VMware configurations, focusing on the creation of an extra virtual disk essential for the {productname} storage partition within VMware environments.
+
+.Procedure: Creating the VMware Virtual Machine
+
+. Download {productname} Server [filename]``.vmdk`` file then transfer a copy to your VMware storage.
+
+. Make a copy of uploaded [filename]``.vmdk`` file using VMware web interface.
+  This will convert provided [filename]``.vmdk`` file to the format suitable for vSpere hypervisor.
+
+. Create and name a new virtual machine based on the Guest OS Family [systemitem]``Linux`` and Guest OS Version {sle} 15 (64-bit).
+
+. Add an additional [systemitem]``Hard Disk 2`` of 500 GB (or more).
+
+. Configure RAM and number of CPUs (at least 16 GB RAM and 4 CPUs).
+
+. Set the network adapter as required.
+
+. Power on the VM, and follow firstboot dialogs.
+
+
+[[minimal.susemgr.prep]]
+== Preparing Virtual Machine for {productname}
+
+Before starting obtain your SUSE Manager Registration Code from SUSE Customer Center - https://scc.suse.com.
+
+
+// FIXME
+// Here we probably want to use sl-micro and thus transactional-update
+// See server-deployment-vm-suma, section "Register {sl-micro} and
+// {productname} {productnumber} Server
+
+[[proc.minimal.susemgr.prep]]
+.Procedure: Preparing for {productname} run
+
+. Log in as `root`.
+
+. Register {productname} with SCC.
+  For example, replace `<productnumber>` with `{productnumber}` and `<architecture>` with `x86_64`:
++
+
+----
+SUSEConnect -e <EMAIL_ADDRESS> -r <SUSE_MANAGER_CODE> \
+  -p SUSE-Manager-Server/<productnumber>/<architecture>
+----
+
+. Validate the authorized extensions by running the list extensions command:
++
+
+----
+SUSEConnect --list-extensions
+----
+
+. Add {productname} repositories:
++
+
+----
+SUSEConnect -p sle-module-basesystem/15.4/x86_64
```

Comment:
This is not correct for 5.0, we don't have 15.4 we have sle micro 5.5 as host.

Reply:
copied from VM chapter.